calm/domains-example/security/schema/micro-segmentation.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://raw.githubusercontent.com/finos/architecture-as-code/main/calm/draft/2024-08/meta/control-requirement.json",
```

Comment:
I don't think this is the correct way to refer to the 'parent' schema . . . this says that micro-segmentation.json is an instance of the schema and expects to have the required properties. I think instead this should be like a pattern when the schema is to the calm.json and it includes a control requirement as a $ref, e.g. https://github.com/finos/architecture-as-code/blob/main/calm/pattern/api-gateway.json - do you think?

Reply:
I agree, there's something wrong with this as it is. Will fix this too, but I think there is a better way to do it.